packages/mail/script/generateTextTemplates.js

```diff
@@ -30,13 +31,59 @@ const run = async () => {
       await fs.writeFile(
         file.replace(/\.html$/gi, '.txt'),
         htmlToText
-          .fromString(html, { ignoreImage: true })
-          .replace(/^ /gm, '')
-          .replace(/kontakt@republik\.ch \[kontakt@republik\.ch\]/gm, 'kontakt@republik.ch'),
+          .fromString(html, {
+            ignoreImage: true,
+            format: {
+              anchor: (elem, fn, options) => {
+                // based on https://github.com/werk85/node-html-to-text/blob/7894809a57e9066501b954dbd9c16fd34c3bcebf/lib/formatter.js#L148-L176
+                // backed in noLinkBrackets: true and hideLinkHrefIfSameAsText: true
+                // custom
+                // - add extra space after urls
+                // - add normalizeUrl for toHideSameLink
+                function getHref () {
+                  if (!elem.attribs || !elem.attribs.href) { return undefined }
+                  const href = elem.attribs.href.replace(/^mailto:/, '')
+                  if (options.noAnchorUrl && href[0] === '#') { return undefined }
+                  return (options.linkHrefBaseUrl && href[0] === '/')
+                    ? options.linkHrefBaseUrl + href
+                    : href
+                }
+                function getText () {
+                  return fn(elem.children, options) || ''
+                }
+
+                const storedCharCount = options.lineCharCount
+                const text = getText()
+                const href = getHref()
+                function normalizeUrl (url = '') {
+                  return url
+                    .replace(/\s+/g, '')
+                    .replace(/^(https?:\/\/)?(www\.)?/, '')
+                    .replace('{{frontend_base_url}}', 'republik.ch')
```

Comment:
why is this needed?

Reply:
to avoid `www.republik.ch {{frontend_base_url}} ` which would result in `www.republik.ch https://www.republik.ch/ `

Now it will just generate `{{frontend_base_url}}` which results in `https://www.republik.ch/ ` (hopefully clickable)